Update Pager.md (#493)
google/accompanist: docs/pager.md

}
```

> If you want to jump to a specific page, for example in a video, after pressing a button to jump to the previous page, you can call the `pagerState.animateScrollToPage(n)` method in a `Coroutine Scope`

Reviewer: ```suggestion
If you want to jump to a specific page, you either call can call `pagerState.scrollToPage(index)` or  `pagerState.animateScrollToPage(index)` method in a `CoroutineScope`.
```
